test/lib/bamboo/adapters/smtp_adapter_test.exs

```diff
@@ -178,7 +178,7 @@ defmodule Bamboo.SMTPAdapterTest do
 
     bamboo_email = new_email()
     bamboo_config = SMTPAdapter.handle_config(configuration(config))
-    :ok = SMTPAdapter.deliver(bamboo_email, bamboo_config)
+    {:ok, _} = SMTPAdapter.deliver(bamboo_email, bamboo_config)
```

Comment:
What do you think of updating the `FakeGenSMTP` module to return some data? It would allow us to test we forward the expected data instead of testing `_`.

Reply:
I'm fine with this. I'm gonna update it.